doc/devel/installation_from_source.rst

```diff
@@ -0,0 +1,270 @@
+.. _installation-from-source:
+
+**********************
+Installing from source
+**********************
+
+Getting the source
+==================
+
+More likely you will want to get the source repository to be able to follow the
+latest changes.  In that case, you can use::
+
+    git clone https://github.com/dipy/dipy.git
+
+For more information about this see :ref:`following-latest`.
+
+After you've cloned the repository, you will have a new directory, containing
+the DIPY ``pyproject.toml`` file, among others.  We'll call this directory - that
+contains the ``pyproject.toml`` file - the *DIPY source root directory*.  Sometimes
+we'll also call it the ``<dipy root>`` directory.
+
+Building and installing
+=======================
+
+Install from source (all operating systems - recommended)
+---------------------------------------------------------
+
+Change directory into the *DIPY source root directory*.
+
+To clean your directory from temporary file, use::
+
+    git clean -fxd
+
+This command will delete all files not present in your github repository.
+
+Then, complete your installation by using this command::
+
+    pip install -r requirements/build.txt
+    pip install --no-build-isolation --user -e .
+
+This command will do the following :
+    - install the requirements for building dipy
+    - remove the old dipy installation if present
+    - build dipy
+    - install dipy locally on your user environment
+
+.. _install-source-nix:
+
+Install from source for Unix (e.g Linux, OSX)
```

Comment:
e.g -> e.g.

OSX: replace all by macOS (they changed the naming/brand a few years ago, 2016)

Reply:
Thanks